lambda_functions/ec2_simple/stop/main.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+from datetime import datetime, timedelta
+
+def is_day_before_patching(date_to_check=None):
+    today = date_to_check or datetime.now().date()
```

Comment:
@Davidoutz would it be possible to pass the information about the date for preventing the stop as an input? This would allow CAS/PAM to easily and independently change the values, if needed.

For ex. 
var.patching_month
var.day_of_the_month